osbrain/tests/test_proxy.py

```diff
@@ -325,3 +329,13 @@ def shoot(self):
     assert not wayne._next_oneway
 
     assert wait_agent_attr(wayne, value=20 * ['bang!'], timeout=1.5)
+
+
+def test_agent_proxy_shutdown_connectionclosed():
```

Comment:
`test_nameserver_proxy_*` instead?

Reply:
Copy-pasting is a hard thing to do well...

Reply:
I know how you feel. :joy: